connectors/src/connectors/microsoft/temporal/activities.ts

```diff
@@ -276,7 +283,8 @@ export async function markNodeAsSeen(connectorId: ModelId, internalId: string) {
   );
 
   if (!node) {
-    throw new Error(`Node ${internalId} not found`);
+    logger.warn(`Node ${internalId} not found`);
```

Comment:
2 coms:
- sure we wanna switch from breaking to logging?
- would log a bit more in that case, at least the connector id; using e.g. a `childLogger` if you're familiar with the pattern?

Reply:
yes, if you remove an item while doing a fullsync , since we don't stop anymore the workflow, it's normal that we fall into this case. The node should just be skipped.

Reply:
Ok clear ! would there be a simple way to detect that? skip if not found *because* item removed, otherwise throw?
Also, is it fine with you to update the logging?